fdbclient/include/fdbclient/FDBTypes.h

```diff
@@ -356,6 +356,10 @@ struct KeyRangeRef {
 		return KeyRangeRef(begin.withPrefix(prefix, arena), end.withPrefix(prefix, arena));
 	}
 
+	static KeyRangeRef entireRangeWithPrefix(const StringRef& prefix) {
+		return KeyRangeRef("/"_sr, "0"_sr).withPrefix(prefix);
+	}
```

Comment:
There is a function here called `prefixRange` which essentially does this, but for arbitrary prefixes (i.e. ones ending in characters besides `/`). In that function, the `/` would be included as part of the prefix.

As a side note, if you do use this function, then it should return a `KeyRange` rather than a `KeyRangeRef` because `withPrefix` is returning a `KeyRange`. Otherwise, you'll end up with a pointer to invalid memory.

Reply:
Thanks for the pointers! I don't need this function anymore (as the newly-tenant-aware API of `getEstimatedRangeSizeBytes` doesn't need the key range to be prefix-ed), so I've removed it altogether.